### DISCUSSION TITLE
Add proxy support for search results (with limiter+redis setup)

### DISCUSSION THREAD

**Discussion by sbstratos79** [searxng/searxng#1115] [Ideas]
I tried setting up searxng with filtron+morty but proxy links weren't working. Always got the morty hash error. Images were also broken. So I switched to limiter+redis and now image proxy is working fine. But there's no option for proxy links. The proxy feature is very useful for bypassing websites that are blocked by isp etc. I know it can easily be circumvented by using a vpn but having a proxy link right there in the search engine is definitely a handy feature. Startpage.com has a similar feature and I find it very useful. 
If morty has been integrated in searxng, isn't there a way to bring back the proxy feature?

**Comment by mrpaulblack** [comment]
No currently there is not. There is already a issue ticket to track progress on bringing back the proxy button feature with redis+limiter plugin stack: https://github.com/searxng/searxng/issues/857
